### PR TITLE
Add EIP-5700: Bindable Token Standard

### PR DESCRIPTION
**Pull request by leeren**
When opening a pull request to submit a new EIP, please use the suggested template: https://github.com/ethereum/EIPs/blob/master/eip-template.md

We have a GitHub bot that automatically merges some PRs. It will merge yours immediately if certain criteria are met:

 - The PR edits only existing draft PRs.
 - The build passes.
 - Your GitHub username or email address is listed in the 'author' header of all affected PRs, inside <triangular brackets>.
 - If matching on email address, the email address is the one publicly listed on your GitHub profile.

### CODE REVIEW

**Comment by leeren** [comment]
@Pandapip1 Seems like the failing tests are unrelated to my EIP changes? What's the right way to diagnose?

File: assets/eip-5700/erc721/ERC721.sol [hunk 1]
```diff
@@ -0,0 +1,240 @@
+// SPDX-License-Identifier: CC0-1.0
+pragma solidity ^0.8.16;
+
+////////////////////////////////////////////////////////////////////////////////
+///              ░▒█▀▀▄░█▀▀█░▒█▀▀█░█▀▀▄░▒█▀▄▀█░▄█░░▒█▄░▒█░▒█▀▀▀              ///
+///              ░▒█░▒█░█▄▀█░▒█▄▄█▒█▄▄█░▒█▒█▒█░░█▒░▒█▒█▒█░▒█▀▀▀              ///
+///              ░▒█▄▄█░█▄▄█░▒█░░░▒█░▒█░▒█░░▒█░▄█▄░▒█░░▀█░▒█▄▄▄              ///
+////////////////////////////////////////////////////////////////////////////////
```
Comment (Pandapip1) [inline assets/eip-5700/erc721/ERC721.sol:8]: Please remove the watermarks.
```suggestion
````

File: assets/eip-5700/erc1155/ERC1155.sol [hunk 2]
```diff
@@ -0,0 +1,200 @@
+// SPDX-License-Identifier: CC0-1.0
+pragma solidity ^0.8.16;
+
+////////////////////////////////////////////////////////////////////////////////
+///              ░▒█▀▀▄░█▀▀█░▒█▀▀█░█▀▀▄░▒█▀▄▀█░▄█░░▒█▄░▒█░▒█▀▀▀              ///
+///              ░▒█░▒█░█▄▀█░▒█▄▄█▒█▄▄█░▒█▒█▒█░░█▒░▒█▒█▒█░▒█▀▀▀              ///
+///              ░▒█▄▄█░█▄▄█░▒█░░░▒█░▒█░▒█░░▒█░▄█▄░▒█░░▀█░▒█▄▄▄              ///
+////////////////////////////////////////////////////////////////////////////////
```
Comment (Pandapip1) [inline assets/eip-5700/erc1155/ERC1155.sol:8]:
```suggestion
```

**Review by Pandapip1 — CHANGES_REQUESTED** [review]

**Comment by leeren** [comment]
@Pandapip1 removed watermarks per your request!

**Comment by leeren** [comment]
Could this please be re-opened?

**Review by Pandapip1 — APPROVED** [review]

**Comment by leeren** [comment]
Will be updating this soon with changes after reflecting on limitations. I've also done a deep analysis on all other EIPs touching on ownership attribution and will add a section on how this differs and the types of compromises being made. Just commenting here to make it clear that this EIP is not dead and a lot of effort will soon be put forth again to asking for feedback!

**Pull request merged**